contracts/SnappAuction.sol

```diff
@@ -111,8 +112,11 @@ contract SnappAuction is SnappBase {
         uint16 accountId = publicKeyToAccountMap(msg.sender);
         require(accountId <= AUCTION_RESERVED_ACCOUNTS, "Accout is not a reserved account");
 
+
         require(packedOrders.length % 26 == 0, "Each order should be packed in 26 bytes!");
         uint numOrders = packedOrders.length / 26;
+        uint32 currentBatchIndex = standingOrders[accountId].currentBatchIndex;
+        require(currentBatchIndex < MAX_CURRENTBATCHINDEX || numOrders == 0, "Last standing order has to be nullifying");
```

Comment:
Should this create a new unit test case instance? What is this really trying to achieve? Do we actually need this?

Also the text doesn't make a lot of sense to me. Nowhere do we check the content of the standing order.

Reply:
Yes, if we decide for the option, I will for sure create an unit test.

if the numOrders == 0, the orders are getting nullified.